jupyter_book/toc.py

```diff
@@ -289,14 +295,12 @@ def _check_toc_entries(sections):
         for key in section.keys():
             if key not in allowed_keys:
                 logger.warning(f"Unknown key in `_toc.yml`: {key}")
-        # Correct for old toc naming
-        # TODO: deprecate in a few release cycles
-        if "url" in section and "path" not in section:
+        if "url" in section and "title" not in section:
```

Comment:
I think we should allow the "old" jupyter book `url:` behavior a bit longer, for at least one release cycle. So let's use the following logic:

* If `url` in section:
  * If `url` starts with "https:"
    * Follow the logic you've added here
  * else:
    * Assume the `url:` is a legacy configuration from old Jupyter Book, and follow the `section["url"].lstrip()` code that you commented out

does that make sense?

Reply:
gotcha, I will do the necessary edits.